fdbclient/FileBackupAgent.actor.cpp

```diff
@@ -2326,6 +2326,12 @@ struct BackupLogsDispatchTask : BackupTaskFuncBase {
 		                                             beginVersion + (CLIENT_KNOBS->BACKUP_MAX_LOG_RANGES - 1) *
 		                                                                CLIENT_KNOBS->LOG_RANGE_BLOCK_SIZE);
 
+		if (endVersion - beginVersion > CLIENT_KNOBS->BACKUP_MAX_VERSION_DIFF) {
+			TraceEvent(SevError, "BackupSawHugeVersionJump")
+			    .detail("Hint", "This may due to DR failover or user operation. Try restart the backup.");
```

Comment:
Better hint:
"This may due to DR failover or user bumping up cluster's version. Try restart the backup."